app/templates/base.html

```diff
@@ -147,7 +57,7 @@
           'text': "Accessibility"
         },
         {
-          'href': url_for('main.cookies'),
+          'href': "#",
```

Comment:
Is there a specific reason for the mix of single and double quotes here? I know I've used them to distinguish HTML from a nested component, but I think we can make it more consistent for most of the contents of this file

Reply:
Changed to single quotes.